src/main/java/org/takes/rq/RqHeaders.java

```diff
@@ -48,7 +48,10 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.1
+ * @todo #342:30min/DEV Extract RqHeaders.Smart into its own file and then
+ *  remove PMD.TooManyMethods suppression
```

Comment:
@erimerturk but we have these `*.Smart` classes inside interfaces, everywhere. We won't extract, no need to put this puzzle here